docs/release.md

```diff
@@ -0,0 +1,55 @@
+# Status AddOn Relesea process 
+
+The Status AddOn release process is based on [GoReleaser](https://goreleaser.com).
+GoReleaser is configured to automatically create a new release by pushing new tags matching the
+pattern 'v*' and build and publish to ghcr.io the packages for the 
+[container image](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon)
+and the [helm chart](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon-chart) 
+used to deploy the status add-on. The relevant files for goreleaser configuration are
+[./goreleaser.yaml](../.goreleaser.yaml) and [./.github/workflows/goreleaser.yml](../.github/workflows/goreleaser.yml).
```

Comment:
That first sentence is very long; it should be broken up. Also, I was taught that the active voice is better than the passive voice. I might reword something like the following.

> The automated part of the process starts with a [GitHub workflow](../.github/workflows/goreleaser.yml) that reacts to a new tag whose name starts with "v" by invoking GoReleaser. A [config file](../.goreleaser.yaml) configures GoReleaser to build and publish the [container image](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon) and the [helm chart](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon-chart), which is used to deploy the status addon.

Reply:
looks good to me. Updated.